1.0/elements/docs-menu.html

```diff
@@ -56,22 +62,24 @@
       <core-item label="Local DOM"><a href="/{{versionPrefix}}/docs/devguide/local-dom.html"></a></core-item>
       <core-item label="Styling"><a href="/{{versionPrefix}}/docs/devguide/styling.html"></a></core-item>
       <core-item label="Events"><a href="/{{versionPrefix}}/docs/devguide/events.html"></a></core-item>
-      <core-submenu label="Data binding" icon="hardware:keyboard-arrow-down" class="guide">
+      <core-submenu label="Data binding" icon="arrow-drop-down" class="guide">
         <core-item label="Syntax"><a href="/{{versionPrefix}}/docs/devguide/data-binding.html"></a></core-item>
         <core-item label="Helper elements"><a href="/{{versionPrefix}}/docs/devguide/templates.html"></a></core-item>
       </core-submenu>
       <core-item label="Behaviors"><a href="/{{versionPrefix}}/docs/devguide/behaviors.html"></a></core-item>
       <core-item label="Utility functions"><a href="/{{versionPrefix}}/docs/devguide/utility-functions.html"></a></core-item>
       <core-item label="Global settings"><a href="/{{versionPrefix}}/docs/devguide/settings.html"></a></core-item>
-      <core-item label="Browser compatibility"><a href="/{{versionPrefix}}/resources/compatibility.html"></a></core-item>
       <core-item label="Experimental features" icon="warning"><a href="/{{versionPrefix}}/docs/devguide/experimental.html"></a></core-item>
     </core-submenu>
 
-
     <core-item label="API Reference" icon="subject">
       <a href="https://polymer.github.io/polymer" target="_blank"></a>
     </core-item>
 
+    <core-item label="Tools" icon="editor:format-paint">
+      <a href="/{{versionPrefix}}/resources/tools.html"></a>
```

Comment:
How would you feel about moving this doc /tools/overview.{html|md}? We have upcoming docs from Rob (vulcanize), Kayce (wct), so we could start out with a real section.